stdlib/experimental/influxdb/influxdb.go

```diff
@@ -151,6 +151,9 @@ func api(ctx context.Context, a values.Object) (values.Value, error) {
 		req.Header.Set("Authorization", "Token "+token)
 	}
 
+	if host == "" {
+		client = fluxhttp.NewPrivateClient(client)
```

Comment:
Instead of this can't you call `deps.PrivateHTTPClient` here?

Reply:
The most recent commit does that. I must have pushed it up after you added this comment.